packages/catalog-model/src/schema/shared/common.schema.json

```diff
@@ -43,6 +43,12 @@
         },
         "target": {
           "$ref": "#reference"
+        },
+        "targetRef": {
+          "type": "string",
+          "minLength": 1,
+          "pattern": "^\\w+$",
```

Comment:
`\w+` isn't quite enough. I think we could skip the pattern in this case though, in practice it's completely open and I don't see us adding patterns in other places.

Reply:
Fixed 👍